content/docs/operating/configuration.md

```diff
@@ -995,16 +990,77 @@ Where `<scheme>` may be `http` or `https` and `<path>` is a valid URL path.
 CAUTION: Remote write is experimental: breaking changes to configuration are
 likely in future releases.
 
-`url` is the URL of the endpoint to send samples to. `remote_timeout` specifies
-the timeout for sending requests to the URL. There are no retries.
-
-`basic_auth`, `tls_config` and `proxy_url` have the same meanings as in a
-`scrape_config`.
-
-`write_relabel_configs` is relabelling applied to samples before sending them
- to the URL. Write relabelling is applied after external labels. This could be
-used to limit which samples are sent.
+`write_relabel_configs` is relabeling applied to samples before sending them
+to the remote endpoint. Write relabeling is applied after external labels. This
+could be used to limit which samples are sent.
 
 There is a [small
 demo](https://github.com/prometheus/prometheus/tree/master/documentation/examples/remote_storage)
 of how to use this functionality.
+
+```
+# The URL of the endpoint to send samples to.
+url: <string>
+
+# Timeout for requests to the remote write endpoint.
+[ remote_timeout: <duration> | default = 30s ]
+
+# List of remote write relabel configurations.
+write_relabel_configs:
+  [ - <relabel_config> ... ]
+
+# Sets the `Authorization` header on every remote write request with the
```

Comment:
I was wondering this too, but the problem is that these are embedded fields in the config, and we'd have to make up another notation to describe that, making our config notation even more complex and confusing. So maybe just copy&paste is the least bad.